.github/workflows/docker.yml

```diff
@@ -104,7 +106,9 @@ jobs:
           context: .
           file: ${{ env.DOCKERFILE_ALPINE }}
           platforms: ${{ env.DOCKER_PLATFORMS }}
-          push: ${{ github.event_name != 'pull_request' && github.actor != 'dependabot[bot]' }}
+          push: ${{ (github.event_name != 'pull_request' &&
+                     github.actor != 'dependabot[bot]') ||
+                     startsWith(github.ref_name, 'release-plz') }}
```

Comment:
Looks good, but is the `ref_name` for `release-plz` documented somewhere? I was wondering if we should check for a tag name instead. 🤔

Reply:
@mre It's documented here: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#github-context, but taking a closer look, it might not work for a pull request.

Reply:
@mre I've refactored this pull request. WDYT?

Reply:
@mre Did you want me to ask another maintainer to review?

Reply:
Oh, sorry for dropping the ball on this. 😅